README.md

```diff
@@ -21,7 +21,7 @@ Types and bindings to connect easily to a running Holochain conductor from Rust.
 
 ``` bash
 ./build-fixture.sh
-cargo test
+cargo test --release
```

Comment:
Running `cargo test` without the `--release` flag causes the tests to fail so I updated the command.

Reply:
@c12i and @jost-s, do you know if you experience the same behaviour? The `build-fixture.sh` script does expect the release build and it seems so does the DNA, but if I change those two to use the debug build then the tests still fail.

Reply:
Okay, thanks. I think this change to the README is still good then.